src/client/pages/OfferNew/index.tsx

```diff
@@ -59,18 +89,54 @@ export const OfferNew: React.FC = () => {
     skip: quoteIdsIsLoading,
   })
 
+  const bundleVariants = React.useMemo(
+    () =>
+      (data?.quoteBundle.possibleVariations as Array<QuoteBundleVariant>) ?? [],
+    [data?.quoteBundle.possibleVariations],
+  )
+  const canShowInsuranceSelector =
+    isInsuranceToggleEnabled && bundleVariants.length > 1
+  const [selectedBundleVariant, setSelectedBundleVariant] = React.useState<
+    QuoteBundleVariant
+  >()
+
+  useEffect(() => {
+    if (!selectedBundleVariant) {
+      return setSelectedBundleVariant(
+        getBundleVariantFromQuoteIds(selectedQuoteIds, bundleVariants),
+      )
```

Comment:
The return value of a `useEffect` should be a clean-up function no? I think it's confusing that we return (void?) here.

Reply:
I used return here just as a `break` statement. I thought `return void` is equivalent to `break`